rf_meta_query/des.py

```diff
@@ -0,0 +1,64 @@
+"""
+To download DES images for a given region
+"""
+
+import numpy as np
+from astropy.coordinates import SkyCoord
+from pyvo.dal import sia
+from astropy import units, io, utils
+import pdb
+
+_DEF_ACCESS_URL = "https://datalab.noao.edu/sia/des_dr1"
+_svc = sia.SIAService(_DEF_ACCESS_URL)
+
+
+# The following function is copied directly from 
```

Comment:
Let's put this info in the main doc string.
These otherwise don't get slurped.

Reply:
Done